R/bugphyzz.R

```diff
@@ -297,30 +299,21 @@ getTaxonSignatures <- function(tax, bp, ...) {
     )
 }
 
-## Import the devel version of bupghyzz
-.downloadDevel <- function(force_download) {
-  types <- c("multistate", "binary", "numeric")
-  urls <- paste0(
-    "https://github.com/waldronlab/bugphyzzExports/raw/main/bugphyzz_",
-    types,
-    ".csv"
-  )
-  names(urls) <- types
-  output <- vector("list", length(urls))
-  for (i in seq_along(output)) {
-    message("Importing ", names(urls)[i], " data...")
-    names(output)[i] <- names(urls)[i]
-    rpath <- .getResource(
-      rname = paste0("bugphyzz_", names(urls)[i], ".tsv"),
-      url = urls[i], verbose = TRUE, force = force_download
-    )
-    output[[i]] <- utils::read.csv(rpath, header = TRUE, skip = 1) |>
-      dplyr::mutate(Attribute = tolower(Attribute))
+## Import a version of bupghyzz
+.downloadResource <- function(version = "d3fd894", force_download) {
```

Comment:
I think we don't want a default version here.

Reply:
Added cff61028a3aece7a7f00aa0998aeb13471dbb241